lms/djangoapps/certificates/models.py

```diff
@@ -998,6 +999,13 @@ class CertificateTemplate(TimeStampedModel):
         help_text=_(u'On/Off switch.'),
         default=False,
     )
+    language = models.CharField(
+        max_length=2,
+        choices=LANGUAGES,
+        blank=True,
```

Comment:
Django admin will submit blank fields with empty string instead of `null`.  Can we confirm which one we want and ensure that that is all that is ever set?

I know @amangano-edx had a similar comment.  Not sure what the resolution was

Reply:
Would all the existing entries from before the table is updated to include this new attribute have the language value initialized to null?
Because in that case we would want to set the empty_value attribute to null instead of its default of the empty string. I'm not sure how column addition works in regards to initialization of values.

Reply:
Yes, existing entries would default to `null`.